src/updaterwindow.cpp

```diff
@@ -294,9 +294,8 @@ void UpdaterWindow::startDownload(const QUrl &url)
     m_startTime = QDateTime::currentDateTime().toSecsSinceEpoch();
     QNetworkRequest netReq(url);
 
-#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
-    netReq.setAttribute(QNetworkRequest::FollowRedirectsAttribute, true);
-#endif
+    netReq.setAttribute(QNetworkRequest::RedirectPolicyAttribute,
+                        QNetworkRequest::NoLessSafeRedirectPolicy);
```

Comment:
Just a note, but that should be the default value for `QNetworkRequest::RedirectPolicyAttribute` according to the documentation, so this line could then be removed entirely.